lib/eunit/vsn.mk

```diff
@@ -1 +1 @@
-EUNIT_VSN = 2.2.9
+EUNIT_VSN = 2.2.10
```

Comment:
Can we be reasonably sure that no tools rely on the older format? If they are, then this is a breaking change and the version should probably reflect that.